src/modules/fw_att_control/FixedwingAttitudeControl.cpp

```diff
@@ -638,6 +673,32 @@ void FixedwingAttitudeControl::run()
 				control_input.groundspeed = groundspeed;
 				control_input.groundspeed_scaler = groundspeed_scaler;
 
+				/* bi-linear interpolation over airspeed for actuator trims */
+				float trim_roll = _parameters.trim_roll;
+				float trim_pitch = _parameters.trim_pitch;
+				float trim_yaw = _parameters.trim_yaw;
+
+				if (airspeed < _parameters.airspeed_trim) {
+					trim_roll = math::gradual(airspeed, _parameters.airspeed_min, _parameters.airspeed_trim,
```

Comment:
It's fine for now, but we should make an effort to do this kind of thing when the actual data updates (orb_update) rather than every iteration.